packages/suite/src/components/suite/FormattedCryptoAmount.tsx

```diff
@@ -35,12 +41,15 @@ interface FormattedCryptoAmountProps {
 export const FormattedCryptoAmount = ({
     value,
     symbol,
+    isBalance,
```

Comment:
In general, I would prefer composition over configuration so I always try to avoid those switches like `isBalance` and `isRawString` and `disableHiddenPlaceholder`, but I find that tolerable here.

Reply:
How would you write it? :)

Reply:
it's possible to split it to multiple components that would share what is common and handle only the differences. So maybe creating some general component that could receive the formatter from more specialised components like FormattedBalance, FormattedRawString, ... but as I said, I'm ok with the current solution here but we should avoid to add more config props to this component in the future.